accounts/migrations/remove_groups.py

```diff
@@ -0,0 +1,26 @@
+from django.db import migrations
+
+
+def forwards_func(apps, schema_editor):
+    Group = apps.get_model("auth", "Group")
+    User = apps.get_model("auth", "User")
+    groups = ["alum", "employee", "faculty", "member", "staff", "student"]
+    db_alias = schema_editor.connection.alias
+
+    Group.objects.bulk_create(Group(name=f"platform_{group}") for group in groups)
+
+    for user in User.objects.all():
+        user_group = user.groups.all()
+        for group in user_group:
+            user.groups.add(f"platfrom_{group}")
```

Comment:
This would result in us adding `platform_` groups for all groups that a user is already a part of, we should limit this to only the platform created groups you have listed above.